hudi-common/src/main/java/org/apache/hudi/common/table/log/block/HoodieHFileDataBlock.java

```diff
@@ -194,7 +194,7 @@ public IndexedRecord next() {
 
     // NOTE: It's important to extend Hadoop configuration here to make sure configuration
     //       is appropriately carried over
-    Configuration inlineConf = new Configuration(blockContentLoc.getHadoopConf());
+    Configuration inlineConf = new Configuration();
```

Comment:
but this wont work. `new Configuration()` will lost everything else set on the driver

Reply:
We have to ensure we use the hadoopConf object passed down from the engine

Reply:
sure. will get to the bottom of it to see why wiring it fails.

Reply:
found the fix. have updated it.